src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2027,6 +2042,133 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
         }
     }
 
+    private function registerSchedulerConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader): void
+    {
+        if (!interface_exists(SchedulerInterface::class)) {
+            throw new LogicException('Scheduler support cannot be enabled as the Scheduler component is not installed. Try running "composer require symfony/scheduler".');
+        }
+
+        $container->setParameter('scheduler.timezone', new \DateTimeZone($config['timezone']));
+        $container->setParameter('scheduler.execution_mode', $config['execution_mode']);
+        $container->setParameter('scheduler.export_directory', $config['export_directory']);
+        $allowedModes = ExecutionModeOrchestratorInterface::EXECUTION_MODES;
```

Comment:
This constant is defined in class and not in the interface

Reply:
Fixed 🙂